tests/test_wave_01.py

```diff
@@ -142,13 +139,11 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
-    
-    raise Exception("Test needs to be completed.")
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
+    assert movie_to_watch in updated_data["watched"]
+    assert movie_to_watch not in updated_data["watchlist"]
+    assert movie_to_watch["title"] == updated_data["watched"][1]["title"]
```

Comment:
The first two assertions here are great! The one on 144 is covered by the one on 142, so you could omit it. If you wanted to , you could also add an assertion for checking which movie is remaining in the watchlist list and another to verify the one already on the watchlist stays there:
```python
    assert FANTASY_1 in updated_data["watchlist"]
    assert FANTASY_2 in updated_data["watched"]
```